app/policies/user_policy.rb

```diff
@@ -1,6 +1,6 @@
 class UserPolicy < ApplicationPolicy
-  DEFAULT_ATTRS = %i[full_name email password address cellphone]
-  ADMIN_ATTRS = %i[admin active]
+  DEFAULT_ATTRS = %i[full_name email password address cellphone].freeze
```

Comment:
I think here you cannot use `freeze`, since you have a `concat` method. I think it's not permitted, can you please check it out?

Reply:
https://apidock.com/ruby/Array/concat

It's permitted because concat method changes the original object. Maybe we should write a unit test for this since this is susceptible to errors.